packages/manager/apps/dedicated/client/app/account/kyc-documents/dashboard/controller.js

```diff
@@ -6,27 +6,37 @@ import {
   LEGAL_FORMS,
   MAXIMUM_DOCUMENTS,
   PRIVACY_LINKS,
+  TRACKING_DOCUMENTS,
+  TRACKING_CTA_SEND_DOCUMENTS,
+  TRACKING_CTA_CONFIRM,
+  TRACKING_CTA_CANCEL,
+  TRACKING_UPLOAD_SUCCESS,
+  TRACKING_UPLOAD_ERROR,
+  TRACKING_CTA_DASHBOARD,
 } from './constants';
 
 import illustration from './assets/picto.svg';
 
 export default class KycDocumentsCtrl {
   /* @ngInject */
-  constructor($translate, $q, $http, coreConfig) {
+  constructor($translate, $q, $http, coreConfig, atInternet) {
     this.$http = $http;
     this.$translate = $translate;
     this.$q = $q;
     this.maximum_size = MAXIMUM_SIZE;
     this.DOCUMENT_TYPE = DOCUMENT_TYPE;
     this.maximum_documents = MAXIMUM_DOCUMENTS;
     this.FRAUD_STATUS = FRAUD_STATUS;
+    this.TRACKING_CTA_DASHBOARD = TRACKING_CTA_DASHBOARD;
     this.illustration = illustration;
     this.privacyLink =
       PRIVACY_LINKS[coreConfig.getUser().ovhSubsidiary] ||
       PRIVACY_LINKS.DEFAULT;
+    this.atInternet = atInternet;
   }
 
   $onInit() {
+    this.atInternet.trackPage({ name: TRACKING_DOCUMENTS });
```

Comment:
No need to do this, you can use the atInternet property on routing to track page. You can check on manager, this already exist like here : https://github.com/ovh/manager/blob/cd6e0c12f8f8e569d00c8f66c19a5ec933f17a2e/packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/servers/server/monitoring/monitoring.routing.js#L17